app/messaging/fetch-application.js

```diff
@@ -8,6 +8,12 @@ const fetchApplication = async (message) => {
     const msgBody = message.body
     console.log('received application fetch request', util.inspect(msgBody, false, null, true))
     const application = await get(msgBody.applicationReference)
+
+    // if application doesn't exists or already submitted return null.
+    if (!application || application?.vetVisit?.dataValues) {
```

Comment:
The same comment as for `process-vet-visit`, can this be extracted to convey meaning please?